appassembler-booter/src/main/java/org/codehaus/mojo/appassembler/booter/AppassemblerBooter.java

```diff
@@ -24,24 +24,22 @@
  * SOFTWARE.
  */
 
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
```

Comment:
Can we keep the code style with explicit imports instead of asterik imports?

Reply:
Sure. I will fix that. I didn't realise it was a thing

Reply:
Okay done.

Reply:
Thanks that makes it easier to read the patches...